modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -440,7 +450,7 @@ public Page<ObjectEntry> getObjectEntries(
 				objectEntryLocalService.getValuesList(
 					groupId, companyId, dtoConverterContext.getUserId(),
 					objectDefinition.getObjectDefinitionId(), predicate, search,
-					start, end, sorts),
+					start, end, sorts, fields),
```

Comment:
Can you inline the variable fields in this usage? Check SF, please.
(since there is only 1 use of the variable, we should inline it)